pkg/client/api/openapi.yaml

```diff
@@ -426,6 +494,16 @@ components:
           $ref: '#/components/schemas/Destination'
       required:
       - destination
+    OrganizationConfiguration:
+      example:
+        companyIdentification: f6eddffd
+      properties:
+        companyIdentification:
```

Comment:
Nit:
I suggest shortening these names:
`OrganizationConfiguration` -> `OrgConfig`
`companyIdentification` -> `companyID`

Reply:
I shortened to OrgConfig. I think it makes sense to keep companyIdentification to differentiate from being an actual ID for a company.

Reply:
Hmm good point, what about `achCompanyID`?

Reply:
We've never called it "Org" in the code, so I'm really confused why we did so here. Elsewhere we said we're keeping the endpoint path the same as Customers (and the original issue), but then made the JSON body a different name? 

I opened https://github.com/moov-io/paygate/pull/608 to address this. We need consistent naming across all services. Clients will be confusing seeing api.moov.io where some calls use `OrgConfig` and others use `OrganizationConfig`. 

I also opened up https://github.com/moovfinancial/engineering-guide/pull/34 to formalize our policy on public model names as we never defined it before.